sorc/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ if (BUILD_UFS)
     "-DNETCDF_DIR=$ENV{NETCDF}"
     "-D32BIT=ON"
     "-DINLINE_POST=ON"
+    "-DMPI=ON"
```

Comment:
Is this supposed to be on for all configurations? Is it on by default already in the model?

Reply:
Removed.